assets/queries/azureResourceManager/not_recommended_log_profile_retention_policy/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "25684eac-daaa-4c2c-94b4-8d2dbb627909",
+  "queryName": "Not Recommended Log Profile Retention Policy",
```

```suggestion
  "queryName": "Unrecommended Log Profile Retention Policy",
```

I think unrecommended looks better, but this is just a suggestion